CHANGELOG.md

```diff
@@ -3,6 +3,19 @@ CHANGELOG
 
 upcoming
 --------
+Recommendation: Breaking change is `login_with_pry` config file - it's probable this will affect people's development or testing locally but shouldn't affect production usage; I'd be comfortable with this being 1.6.0, or bumping to 2.0.0 depending on @arangamani's thoughts.
```

Comment:
There are a few that I can consider as candidates (purely because I like them and think they're cool :D ), but they will need to be tested and merged.  I'll add this to my to-do for next week.

Reply:
After a brief skim of open PRs:
- I'd like to get #166 in, this line might be breaking because it changes what gets returned when creating a dumb slave, but I can look more closely to see if Jenkins is already returning true/false: https://github.com/arangamani/jenkins_api_client/pull/166/files#diff-e6faa48389a58b9287e126ac91aac0c7L259
- #190, #196, #199, #203, #205, #214, #237 are non-breaking and could be added as new functionality (they may need some documentation updates)

- #204 is definitely breaking and is important to add
- #206, #216 are bugfixes; if people have local workarounds they may be "breaking", but are worth adding

I'm not sure if #225 and #256 are ready; I'm also not sure if Groovy script is the right way to handle this, so I would like to leave those out for a later release.

Basically, given my choice, I'd like to incorporate everything except #225 and #256 if we're going to do a version update.  We can stop accepting PRs for the 2.0.0 release, and do it this way.  There are also a lot of unaddressed issues that I haven't had a chance to comment on yet; if you want more breaking changes and a bigger "strategy" I can spend some more time evaluating this and come up with a bigger release.  @arangamani - let me know what you think.

Reply:
@arangamani - pinging again for visibility.

Reply:
@katelovescode Let's go with your proposal and bump version and include only the well-understood PRs.

Reply:
@arangamani - I will review the open PRs listed above and merge them in if they work.  I'll update the changelog.  What's the process for a version bump?  Make a release in git tagged with the new version number and push to develop and master?

Reply:
Yeah, merge everything to master/develop, and then make a tag after all tests pass. Let me know when you have the tag ready, and I'll push it to rubygems.org. That part is still not automated yet.

Reply:
Thanks!  Will do!